concat-on-disk.ipynb

```diff
@@ -0,0 +1,855 @@
+{
```

Comment:
<p>this is only very slightly more memory</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/anndata-tutorials/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>But the total sizes are different since we can concatenate more with unaligned arrays the unaligned has more files. However I am still looking into why it uses so much when aligned.</p>